css/main.css

```diff
@@ -40,6 +45,8 @@ h2,
 h3,
 h4 {
   text-align: center;
+  color: #dee3e3;
```

Comment:
Too bright, use colors from the Solarized theme

Reply:
i refuse

Reply:
Darkened the titles. This is an excellent compromise. The user experience is now even better than before. Honestly i feel pretty great about this

Reply:
You can see how this looks here https://github.com/Mordi490/mordi490.github.io/pull/7#discussion_r641864601